3-Authorization-II/protect-web-api/AppCreationScripts/README_STEPS.md

```diff
@@ -0,0 +1,471 @@
+---
+page_type: sample
+services: ms-identity
+client: Java Spring Web application
+service: Java Spring Web API
+level: 100
+languages:
+ - java
+products:
+ - java
+ - spring-boot
+ - azure-active-directory
+ - msal-java
+platform: Java
+endpoint: AAD v2.0
+urlFragment: ms-identity-java-spring-tutorial
+name: Enable your Java Spring Boot web app to sign in users on your Azure Active Directory and call a protected Web api
+description: Enable your Java Spring Boot web app to sign in users in your Azure AD tenant and call a protected Web api protected with Spring Security
+extendedZipContent: <FILES_OR_FOLDERS_WITH_TWO_ABSOLUTE_PATHS_TO_INCLUDE_WITH_ZIP:PATH(NAME_IN_THE_REPO), TARGET(NAME_IN_THE_ZIP)>
+extensions: <ENTER_CONTENT_THAT_OTHER_TEAMS_CAN_USE_TO_IDENTIFY_SAMPLES>
```

Comment:
Probably need to either fill out these defaults about zip folders and extensions or just get rid of them.

Reply:
Yeah this section needs some revision. Check out the [PR here](https://github.com/Azure-Samples/GenerateScriptsForSample/pull/46), which generates this section correctly.